db/migrate/20240614072648_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
user_id のインデックスは貼らないようにするには
```suggestion
      t.references :user, null: false, foreign_key: true, index: false
```